tests/test_run.py

```diff
@@ -441,6 +441,57 @@ def test_annual_virtual_storage_with_dynamic_cost():
     assert_allclose(rec.data[3], 5)
 
 
+def test_seasonal_virtual_storage():
```

Comment:
Can you put these tests in a class for organisational purposes.

I think we need to also test the following:

- Case with a licence ending before the reset date (i.e. a licence crossing two years).
- The value of a non-depleted licence outside of the active period. I suspect that the licence's volume is still lowered even though it is not active. It should remain unchanged outside the active period?

Reply:
Done. The volume of the node is unchanged outside of the active period